services/app-web/src/testHelpers/jestHelpers.tsx

```diff
@@ -45,23 +74,44 @@ const renderWithProviders = (
         authProvider = {},
         s3Provider = undefined,
         location = undefined,
+        featureFlags = undefined,
     } = options || {}
 
     const { route } = routerProvider
     const s3Client: S3ClientT = s3Provider ?? testS3Client()
     const user = userEvent.setup()
 
+    const flags: FeatureFlagSettings = {
+        ...getDefaultFeatureFlags(),
+        ...featureFlags,
+    }
+
+    const ldProviderConfig: ProviderConfig = {
+        clientSideID: 'test-url',
+        options: {
+            bootstrap: flags,
+            baseUrl: 'test-url',
+            streamUrl: 'test-url',
+            eventsUrl: 'test-url',
+        },
+        ldClient: ldClientMock(flags),
+    }
+
     const renderResult = render(
-        <MockedProvider {...apolloProvider}>
-            <MemoryRouter initialEntries={[route || '']}>
-                <AuthProvider authMode={'AWS_COGNITO'} {...authProvider}>
-                    <S3Provider client={s3Client}>
-                        {location && <WithLocation setLocation={location} />}
-                        <PageProvider>{ui}</PageProvider>
-                    </S3Provider>
-                </AuthProvider>
-            </MemoryRouter>
-        </MockedProvider>
+        <LDProvider {...ldProviderConfig}>
```

Comment:
Should we be using LDProvider in App.tsx? I'm a little confused how these get read correctly, but it seems like a major improvement in how LD works

Reply:
> Should we be using LDProvider in App.tsx?
We actually initialize LaunchDarkly in `app-web/src/index.tsx` using `LDProvider`.

The important part of this method is that instead of letting `LDProvider` initialize `ldClient` by making the API calls to get the flags, we initialize it ourselves with predefined flags and then pass it to `LDProvider`.